example/config.yml

```diff
@@ -4,6 +4,12 @@ backend:
 
 media_folder: "assets/uploads"
 
+dynamic_widgets:
```

Comment:
Probably better to keep widget configuration on the widget itself, even if it's redundant. I'd suggest accepting the list of widgets under a `widgets` key on the dynamic widget config.